test/compiler/converter_0_5_test.exs

```diff
@@ -107,28 +107,28 @@ defmodule Surface.Compiler.Converter_0_5Test do
            """
   end
 
-  ## Planned changes. Uncomment as the related implementation gets merged
+  test "convert <If> into <#if>" do
+    expected =
+      convert("""
+      <div>
+        <If condition={{ @var }}>
+          1
+          </If>
+        <If   condition={{@var}}>2</If>
+      </div>
+      """)
 
-  # test "convert <If> into <#if>" do
-  #   expected =
-  #     convert("""
-  #     <div>
-  #       <If condition={{ @var }}>
-  #         1
-  #         </If>
-  #       <If   condition={{@var}}>2</If>
-  #     </div>
-  #     """)
+    assert expected == """
+           <div>
+             <#if condition={@var}>
```

Comment:
should we also automatically switch this to using the root prop? (i.e. `<#if {@var}>`)

Reply:
That would not be possible with the current implementation of the converter. Since this is also valid syntax, I think we can keep it as it is for now. The `surface_formatter` on the other hand, can do this quite well.